src/main/webapp/index.jsp

```diff
@@ -13,7 +13,7 @@
     </jsp:attribute>
 
     <jsp:body>
-
+        <a href="bmiCalculator.jsp">BMI Calculator</a>
```

Comment:
Denne her skal vel fjernes inden i comitter. Tænker i har brugt det for at tilgå jeres side undervejs.